conf/etc/v2/controller_interface.json

```diff
@@ -1,6 +1,6 @@
 {
     "HW": {
-        "corosync-pacemaker": [
+        "corosync": [
```

Comment:
Why are we changing it? If this is required then "pacemaker" is better word?

Reply:
Handled in other PR https://github.com/Seagate/cortx-ha/pull/356